### DISCUSSION TITLE
Which minimum @mantine/core (and others packages) version to work with React v17.x?

### DISCUSSION THREAD

**Discussion by koddr** [mantinedev/mantine#2764] [Q&A] [answered]
Hi,

I've one question about the version of Mantine: which minimum version to work with React `v17.x`?

**Comment by rtivital** [comment, marked as answer]
All versions of mantine supports react 17, if you experience issues with it you can open an issue.

**Reply by koddr** [reply]
Thanks for reply, @rtivital.

If I understand you correctly, then Mantine version `5.6.0` (latest) should work correctly with React version [17.0.2 (March 22, 2021)](https://github.com/facebook/react/releases/tag/v17.0.2)?

**Reply by rtivital** [reply]
Yes, but there is one reported issue, the compatibility will be fixed in 5.6.1
